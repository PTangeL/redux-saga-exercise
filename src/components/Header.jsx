import React from 'react'

const Header = () => (
  <header>
    <div className="container">
      <div className="row">
        <div className="col-lg-12">
          <img className="img-responsive logo-academy" role="presentation" src="/public/img/logo.png" />
          <div className="intro-text">
            <span className="name">Beer Game</span>
            <hr className="star-light" />
          </div>
        </div>
      </div>
    </div>
  </header>
)

export default Header
