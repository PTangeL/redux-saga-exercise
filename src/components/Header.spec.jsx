import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'

import Header from './Header'

describe('<Header />', () => {
  it('should render component', () => {
    //TODO Implement a snapshot test
  })
})
