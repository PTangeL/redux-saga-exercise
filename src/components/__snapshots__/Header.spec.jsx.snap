// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render component 1`] = `
<header>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-lg-12"
      >
        <img
          className="img-responsive logo-academy"
          role="presentation"
          src="/public/img/logo.png"
        />
        <div
          className="intro-text"
        >
          <span
            className="name"
          >
            Tinder Beer
          </span>
          <hr
            className="star-light"
          />
        </div>
      </div>
    </div>
  </div>
</header>
`;
